amazon_payments/__init__.py

```diff
@@ -1,3 +1,12 @@
-from api import AmazonPaymentsAPI, AmazonPaymentsAPIError  # noqa
+from pkg_resources import get_distribution
+
+__version__ = get_distribution('django-oscar-amazon-payments').version
+
+VERSION = __version__
+
+try:
```

Comment:
@monokrome @pdufour the try might be overkill at this point as there won't be errors related to the import at setup